### PR TITLE
feat(InputSelect): migrate to tailwind

### PR DESCRIPTION
**Pull request by vbulant**
Closes [FEPLT-1765](https://kiwicom.atlassian.net/browse/FEPLT-1765)

# This Pull Request meets the following criteria:

### For Tailwind migrations:

- [x] Visual regression test suite has been added if does not exist.
- [x] `tailwind-migration-status.yaml` file has been updated with the migrated component(s).

### For new components:

- [ ] Tests have been added/adjusted for my new feature
- [ ] New Components are registered in index.js of my project
- [ ] New Components have `d.ts` files and are exported in `index.d.ts`

[FEPLT-1765]: https://kiwicom.atlassian.net/browse/FEPLT-1765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/orbit-components/src/InputSelect/index.tsx [hunk 1]
```diff
@@ -399,74 +387,104 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
       );
 
     const dropdown = isOpened && (
-      <StyledDropdown
+      <ul
+        className={cx(
+          "font-base bg-white-normal lm:absolute lm:inset-x-0 lm:overflow-y-scroll lm:shadow-action lm:rounded-normal z-[3] flex w-full flex-col",
```
Comment (DSil) [inline packages/orbit-components/src/InputSelect/index.tsx:392]: I think we should enforce `list-none` here, WDYT?

File: packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    <div className="p-md min-h-[600px]">
+      <InputSelect options={simpleOptions} dataTest="Dropdown" maxWidth="200px" maxHeight="200px" />
+    </div>
+  );
+}
+
+export function InputSelectDropdownEmptyStory() {
+  return (
+    <div className="p-md min-h-[600px]">
+      <InputSelect options={[]} dataTest="Dropdown" emptyState="I ain’t no Pokémon" />
```
Comment (DSil) [inline packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx:103]:
```suggestion
      <InputSelect options={[]} dataTest="Dropdown" emptyState="I ain't no Pokémon" />
```

**Review by DSil — COMMENTED** [review]
Great work 💪 just two small 💅

File: packages/orbit-components/src/InputSelect/index.tsx (hunk 1 above)
Comment (vbulant) [reply packages/orbit-components/src/InputSelect/index.tsx:392]: Even though it is already present in the tailwind preflight? I was thinking about it but in the end decided to rely just on the preflight to keep the component styles leaner.

File: packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx (hunk 2 above)
Comment (vbulant) [reply packages/orbit-components/src/InputSelect/InputSelect.ct-story.tsx]: ✅

File: packages/orbit-components/src/InputSelect/index.tsx (hunk 1 above)
Comment (DSil) [reply packages/orbit-components/src/InputSelect/index.tsx:392]: Ah, my bad, I did not notice it was on preflight. No needed then! Thanks

**Review by DSil — APPROVED** [review]

**Pull request merged**